### PR TITLE
Change log for VIPCS 2.2.0

### PR DESCRIPTION
**Pull request by GaryJones**
❗ Reviews & approval welcome. I'll merge this myself (if approved) just before the release. ❗

Includes all currently merged changes.

Since we now have an external contributor (Juliette), then we can/should start giving props.

I also find it useful when other change logs give a link to the PR where I can start following the trail of what exactly the change was and why it was made, so I've added them in as well. By using full inline markdown link (rather than just `[#123]` inline and the `[#123]: https://…` at the bottom), then it makes the copy-paste into the Release notes cleaner.

### CODE REVIEW

**Review by rebeccahum — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
...
+- [#558](https://github.com/Automattic/VIP-Coding-Standards/pull/558): Test bootstrap: various minor tweaks.
+- [#571](https://github.com/Automattic/VIP-Coding-Standards/pull/571): CS: change yoda conditions to non-yoda.
+- [#573](https://github.com/Automattic/VIP-Coding-Standards/pull/573): Composer: Change minimum stability to stable.
+
+### Fixed
+- [#503](https://github.com/Automattic/VIP-Coding-Standards/pull/503): RulesetTest, fix compatibility with Windows.
+- [#504](https://github.com/Automattic/VIP-Coding-Standards/pull/504): RulesetTest: fail the build on failing ruleset tests, fix the failing ruleset test, and fix the test script to handle 0 values.
+- [#505](https://github.com/Automattic/VIP-Coding-Standards/pull/505): DeclarationCompatibility: fix incorrect signature check for `Walker::walk()`.
+- [#509](https://github.com/Automattic/VIP-Coding-Standards/pull/509): RulesetTest: Revert #485 and fix one of the three causes properly.
+- [#557](https://github.com/Automattic/VIP-Coding-Standards/pull/557): Contributing: remove duplicate info.
```
Comment (jrfnl) [inline CHANGELOG.md:32]: This was introduced in #501, so can probably be lumped in with that changelog entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- [#494](https://github.com/Automattic/VIP-Coding-Standards/pull/494): `.gitattributes` file.
+- [#495](https://github.com/Automattic/VIP-Coding-Standards/pull/495): `CODEOWNERS` file.
+- [#450](https://github.com/Automattic/VIP-Coding-Standards/pull/450): [VariableAnalysis](https://github.com/sirbrillig/phpcs-variable-analysis/) package.
+- [#560](https://github.com/Automattic/VIP-Coding-Standards/pull/560): Allow checking test code coverage.
+- [#579](https://github.com/Automattic/VIP-Coding-Standards/pull/560): Docs: Add comparisons and props to change log for old versions.
+
+### Changed
+- [#500](https://github.com/Automattic/VIP-Coding-Standards/pull/500): Travis: change from "trusty" to "xenial".
+- [#501](https://github.com/Automattic/VIP-Coding-Standards/pull/501): Move and improve `CONTRUBUTING.md`.
```
Comment (jrfnl) [inline CHANGELOG.md:20]:
```suggestion
- [#501](https://github.com/Automattic/VIP-Coding-Standards/pull/501): Move and improve `CONTRIBUTING.md`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2020-09-09
+
+Props: GaryJones, jrfnl, rebeccahum.
+
+### Added
+- [#494](https://github.com/Automattic/VIP-Coding-Standards/pull/494): `.gitattributes` file.
+- [#495](https://github.com/Automattic/VIP-Coding-Standards/pull/495): `CODEOWNERS` file.
+- [#450](https://github.com/Automattic/VIP-Coding-Standards/pull/450): [VariableAnalysis](https://github.com/sirbrillig/phpcs-variable-analysis/) package.
```
Comment (jrfnl) [inline CHANGELOG.md:14]: Should a "Deprecated" header be added with an entry that the old VIPCS version of this sniff is now deprecated and will be removed in the next major ?

**Review by jrfnl — CHANGES_REQUESTED** [review]
Generally looking good.

Aside from the inline notes, two additional remarks:
* An entry is missing for PR #561 / issue #469.
* Most entries seem to use the PR title. While this may be clear for people reasonably deeply into the sniffs, I wonder if some can be made more descriptive to allow end-users to understand the change better.
    May not be needed if this changelog is mainly intended for internal use, not for the customers.

File: CHANGELOG.md (hunk 3 above)
Comment (GaryJones) [reply CHANGELOG.md:14]: Added.

File: CHANGELOG.md (hunk 2 above)
Comment (GaryJones) [reply CHANGELOG.md:20]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (GaryJones) [reply CHANGELOG.md:32]: Deleted.

File: CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (GaryJones) [reply CHANGELOG.md:14]: Done.

**Pull request merged**